build.gradle

```diff
@@ -53,15 +53,30 @@ allprojects {
 }
 
 jar {
```

Comment:
I have a quick question about this:

The library have some dependence, like:

- JIconFont.

I don't have more experience about java9 module and now my question is:

How does maven know that need other dependence if inside the jar there isn't any referment to this project? No module-info.java

Reply:
When adding the dependency in pom.xml / build.gradle, build tool downloads the pom file of the file and the jar. POM contains information about dependencies which in turn have the poms as well. This way all the transitive dependencies are fetched and added to the classpath during build / runtime.

Reply:
Hi @metteo, thank you I will lose this part.

This is a great update at the library.

P.S: This is my fault sorry